client/question/components/LearnerViewDirective.js

```diff
@@ -1310,6 +1310,17 @@ tie.directive('learnerView', [function() {
             $scope.privacyModalIsDisplayed = true;
           }
         }
+
+        /**
+         * Refreshes UI if the window loads at 0 height or width (IFrame Firefox
+           Case)
+         */
+        var millisecondDelay = 500;
+        $interval(function() {
+          if ($window.innerWidth === 0 || $window.innerHeight === 0) {
+            codemirrorEditorInstance.refresh();
+          }
+        }, millisecondDelay, 1);
```

Comment:
Is the idea to run this only once? If so, then $timeout might be better.

On the other hand, if the idea is to run this multiple times until the condition holds, then you can use $interval but then the $interval should be cancelled once the conditions are successful.

Reply:
Done. Replaced with $timeout, equally effective.